moped-etl/data-tracker-sync/data_tracker_sync.py

```diff
@@ -0,0 +1,198 @@
+#!/usr/bin/env python
+
+import os
+import argparse
+import logging
+from datetime import datetime, timezone
+
+import knackpy
+
+from process.request import make_hasura_request
+from process.logging import get_logger
+from process.knack_data import build_knack_project_from_moped_project
+from process.queries import (
+    GET_SYNCED_PROJECTS,
+    GET_TEST_SYNCED_PROJECTS,
+    GET_UNSYNCED_PROJECTS,
+    GET_TEST_UNSYNCED_PROJECTS,
+    UPDATE_MOPED_PROJECT_KNACK_ID,
+)
+
+KNACK_DATA_TRACKER_APP_ID = os.getenv("KNACK_DATA_TRACKER_APP_ID")
+KNACK_DATA_TRACKER_API_KEY = os.getenv("KNACK_DATA_TRACKER_API_KEY")
+TEST_MOPED_PROJECT_ID = os.getenv("TEST_MOPED_PROJECT_ID")
+
+KNACK_DATA_TRACKER_PROJECT_OBJECT = "object_201"
+
+
+def find_unsynced_moped_projects(is_test=False):
```

Comment:
I'd really appreciate a brief comment that explains what an unsynced moped project is, like "returns list of moped projects that do not have knack project ids associated with them", i know we can check the queries but....

Reply:
I like this a lot! I agree that it is hard to link the create/sync/update/unsync usage and that won't get easier with time. I'll add some comments. 🙏

Reply:
thanks!!